src/libraries/System.Security.Cryptography.Cose/src/System.Security.Cryptography.Cose.csproj

```diff
@@ -7,9 +8,16 @@
   </PropertyGroup>
 
   <ItemGroup>
-    <Compile Include="$(CommonPath)System\HashCodeRandomization.cs" Link="Common\System\HashCodeRandomization.cs" />
-    <Compile Include="$(CommonPath)System\Memory\PointerMemoryManager.cs" Link="Common\System\Memory\PointerMemoryManager.cs" />
-    <Compile Include="$(LibrariesProjectRoot)System.Formats.Cbor\src\System\Formats\Cbor\CborInitialByte.cs" Link="System\Formats\Cbor\CborInitialByte.cs" />
+    <Compile Include="$(CommonPath)System\HashCodeRandomization.cs"
+             Link="Common\System\HashCodeRandomization.cs" />
+    <Compile Include="$(CommonPath)System\Memory\PointerMemoryManager.cs"
+             Link="Common\System\Memory\PointerMemoryManager.cs" />
+    <Compile Include="$(CommonPath)System\Security\Cryptography\IncrementalHash.netfx.cs"
```

Comment:
S.S.C.Cose uses `IncrementalHash` all over the place, so polyfilling it is probably the best thing to do here.